tmt/steps/report/reportportal.py

```diff
@@ -436,13 +426,57 @@ def append_description(self, curr_description: str) -> str:
                 curr_description = self.data.launch_description
         return curr_description
 
+    def upload_result_logs(
+            self,
+            result: tmt.result.BaseResult,
+            session: requests.Session,
+            item_uuid: str,
+            launch_uuid: str,
+            timestamp: str,
+            write_out_failures: bool = True) -> None:
+        """ Upload all result log files into the ReportPortal instance """
+
+        for index, log_path in enumerate(result.log):
+            try:
+                log = self.step.plan.execute.read(log_path)
+            except tmt.utils.FileError:
+                continue
+
+            level = "INFO" if log_path == result.log[0] else "TRACE"
```

Comment:
Or `index == 0`?

Reply:
I think checking the index should also be fine. Like this?
- https://github.com/teemtee/tmt/pull/3331/commits/a65541e9cce8ff09b65e262762e035fb87deb4b8

Thanks